Dockerfile

```diff
@@ -282,6 +282,8 @@ RUN find $HOME -type d -exec chmod go=u {} + && \
 
 ENV IS_DOCKER_8395080871=1
 
+# ABI tags can interfere when running on Singularity
+RUN strip --remove-section=.note.ABI-tag /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
```

Comment:
<details>
<summary>Docker wb_command ldd</summary>

```
	linux-vdso.so.1 (0x00007ffc7652b000)
	libftgl.so.2 => /lib/x86_64-linux-gnu/libftgl.so.2 (0x00007f596acc0000)
	libQt5Gui.so.5 => /lib/x86_64-linux-gnu/libQt5Gui.so.5 (0x00007f596a6d5000)
	libQt5Network.so.5 => /lib/x86_64-linux-gnu/libQt5Network.so.5 (0x00007f596a514000)
	libQt5Xml.so.5 => /lib/x86_64-linux-gnu/libQt5Xml.so.5 (0x00007f596a4d2000)
	libOSMesa.so.8 => /lib/x86_64-linux-gnu/libOSMesa.so.8 (0x00007f59698cd000)
	libGLU.so.1 => /lib/x86_64-linux-gnu/libGLU.so.1 (0x00007f596985b000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f596983d000)
	libQt5Core.so.5 => /lib/x86_64-linux-gnu/libQt5Core.so.5 (0x00007f59692f4000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f5969112000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f5968fc3000)
	libgomp.so.1 => /lib/x86_64-linux-gnu/libgomp.so.1 (0x00007f5968f81000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f5968f66000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f5968d72000)
	libfreetype.so.6 => /lib/x86_64-linux-gnu/libfreetype.so.6 (0x00007f5968cb3000)
	libGL.so.1 => /lib/x86_64-linux-gnu/libGL.so.1 (0x00007f5968c2b000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f5968c08000)
	libpng16.so.16 => /lib/x86_64-linux-gnu/libpng16.so.16 (0x00007f5968bd0000)
	libharfbuzz.so.0 => /lib/x86_64-linux-gnu/libharfbuzz.so.0 (0x00007f5968acb000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f5968ac3000)
	libglapi.so.0 => /lib/x86_64-linux-gnu/libglapi.so.0 (0x00007f5968a89000)
	libLLVM-12.so.1 => /lib/x86_64-linux-gnu/libLLVM-12.so.1 (0x00007f59632d3000)
	libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f596322a000)
	libicui18n.so.66 => /lib/x86_64-linux-gnu/libicui18n.so.66 (0x00007f5962f2b000)
	libicuuc.so.66 => /lib/x86_64-linux-gnu/libicuuc.so.66 (0x00007f5962d43000)
	libpcre2-16.so.0 => /lib/x86_64-linux-gnu/libpcre2-16.so.0 (0x00007f5962cc0000)
	libdouble-conversion.so.3 => /lib/x86_64-linux-gnu/libdouble-conversion.so.3 (0x00007f5962caa000)
	libglib-2.0.so.0 => /lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f5962b81000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f596be63000)
	libGLdispatch.so.0 => /lib/x86_64-linux-gnu/libGLdispatch.so.0 (0x00007f5962ac9000)
	libGLX.so.0 => /lib/x86_64-linux-gnu/libGLX.so.0 (0x00007f5962a93000)
	libgraphite2.so.3 => /lib/x86_64-linux-gnu/libgraphite2.so.3 (0x00007f5962a66000)
	libffi.so.7 => /lib/x86_64-linux-gnu/libffi.so.7 (0x00007f5962a5a000)
	libedit.so.2 => /lib/x86_64-linux-gnu/libedit.so.2 (0x00007f5962a22000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f5962a17000)
	libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007f59629e5000)
	libicudata.so.66 => /lib/x86_64-linux-gnu/libicudata.so.66 (0x00007f5960f24000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f5960eb1000)
	libX11.so.6 => /lib/x86_64-linux-gnu/libX11.so.6 (0x00007f5960d74000)
	libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f5960d5a000)
	libxcb.so.1 => /lib/x86_64-linux-gnu/libxcb.so.1 (0x00007f5960d2e000)
	libXau.so.6 => /lib/x86_64-linux-gnu/libXau.so.6 (0x00007f5960d28000)
	libXdmcp.so.6 => /lib/x86_64-linux-gnu/libXdmcp.so.6 (0x00007f5960d20000)
```
</details>

<details>
<summary>Singularity wb_command ldd</summary>

```
linux-vdso.so.1 (0x00007ffecaf97000)
	libftgl.so.2 => /lib/x86_64-linux-gnu/libftgl.so.2 (0x00007f6f619e9000)
	libQt5Gui.so.5 => /lib/x86_64-linux-gnu/libQt5Gui.so.5 (0x00007f6f613fe000)
	libQt5Network.so.5 => /lib/x86_64-linux-gnu/libQt5Network.so.5 (0x00007f6f6123d000)
	libQt5Xml.so.5 => /lib/x86_64-linux-gnu/libQt5Xml.so.5 (0x00007f6f611fb000)
	libOSMesa.so.8 => /lib/x86_64-linux-gnu/libOSMesa.so.8 (0x00007f6f60703000)
	libGLU.so.1 => /lib/x86_64-linux-gnu/libGLU.so.1 (0x00007f6f60691000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f6f60673000)
	libQt5Core.so.5 => not found
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f6f60491000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f6f60342000)
	libgomp.so.1 => /lib/x86_64-linux-gnu/libgomp.so.1 (0x00007f6f60300000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f6f602e5000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f6f600f1000)
	libfreetype.so.6 => /lib/x86_64-linux-gnu/libfreetype.so.6 (0x00007f6f60032000)
	libGL.so.1 => /lib/x86_64-linux-gnu/libGL.so.1 (0x00007f6f5ffaa000)
	libQt5Core.so.5 => not found
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f6f5ff87000)
	libpng16.so.16 => /lib/x86_64-linux-gnu/libpng16.so.16 (0x00007f6f5ff4f000)
	libharfbuzz.so.0 => /lib/x86_64-linux-gnu/libharfbuzz.so.0 (0x00007f6f5fe48000)
	libQt5Core.so.5 => not found
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f6f5fe42000)
	libQt5Core.so.5 => not found
	libglapi.so.0 => /lib/x86_64-linux-gnu/libglapi.so.0 (0x00007f6f5fe09000)
	libLLVM-11.so.1 => /lib/x86_64-linux-gnu/libLLVM-11.so.1 (0x00007f6f5ae63000)
	libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f6f5adba000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f6f62b8c000)
	libGLdispatch.so.0 => /lib/x86_64-linux-gnu/libGLdispatch.so.0 (0x00007f6f5ad02000)
	libGLX.so.0 => /lib/x86_64-linux-gnu/libGLX.so.0 (0x00007f6f5acce000)
	libglib-2.0.so.0 => /lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f6f5aba5000)
	libgraphite2.so.3 => /lib/x86_64-linux-gnu/libgraphite2.so.3 (0x00007f6f5ab76000)
	libffi.so.7 => /lib/x86_64-linux-gnu/libffi.so.7 (0x00007f6f5ab6a000)
	libedit.so.2 => /lib/x86_64-linux-gnu/libedit.so.2 (0x00007f6f5ab32000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f6f5ab27000)
	libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007f6f5aaf7000)
	libX11.so.6 => /lib/x86_64-linux-gnu/libX11.so.6 (0x00007f6f5a9b8000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f6f5a945000)
	libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f6f5a92b000)
	libxcb.so.1 => /lib/x86_64-linux-gnu/libxcb.so.1 (0x00007f6f5a901000)
	libXau.so.6 => /lib/x86_64-linux-gnu/libXau.so.6 (0x00007f6f5a8fb000)
	libXdmcp.so.6 => /lib/x86_64-linux-gnu/libXdmcp.so.6 (0x00007f6f5a8f1000)
```
</details>

Should this be 
```suggestion
RUN strip --remove-section=.note.ABI-tag /lib/x86_64-linux-gnu/libQt5Core.so.5
```
?

Reply:
Looks like `/lib` is a symlink to `/usr/lib`.